storybook/stories/docs/introduction.mdx

```diff
@@ -28,7 +29,7 @@ The site shows the individual components in the sidebar and the Canvas on the ri
 
 To view the documentation for each component use the **Docs** menu item in the top toolbar.
 
-To view the source code for the component and its stories on GitHub, click the <InlineIcon icon="repository" /> View Source Repository button in the top right corner.
+To view the source code for the component and its stories on GitHub, click the <InlineIcon icon={ RepoIcon } /> View Source Repository button in the top right corner.
```

Comment:
We're using `RepoIcon` from `@storybook/icons` directly since `Icon` from `@storybook/components` is deprecated, see https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#icons-is-deprecated